s/scipy/scipy_ubi_9.3_v1.10.1.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : scipy
+# Version       : v1.10.1
+# Source repo   : https://github.com/scipy/scipy
+# Tested on     : UBI 9.3
+# Language      : Python, C, Fortran, C++, Cython, Meson
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Puneet Sharma <Puneet.Sharma21@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=scipy
+PACKAGE_VERSION=${1:-v1.10.1}
+PACKAGE_URL=https://github.com/scipy/scipy
+
+OS_NAME=`cat /etc/os-release | grep "PRETTY" | awk -F '=' '{print $2}'`
+
+# install core dependencies
+yum install -y gcc gcc-c++ gcc-fortran pkg-config openblas-devel python3.11 python3.11-pip python3.11-devel git atlas
+
+# install scipy dependency(numpy wheel gets built and installed) and build-setup dependencies
+python3.11 -m pip install meson ninja 'numpy<1.23' 'setuptools<60.0' Cython==0.29.37
```

Comment:
Just use `python` or `pip` instead of `python3` or `pip3`

Reply:
Hi @shubham-dayma-ibm , in existing build scripts, using same **python3.11** and **python3.11 -m pip install**